supply/lib/supply/uploader.rb

```diff
@@ -89,8 +89,8 @@ def perform_upload_meta(version_codes, track_name)
           UI.user_error!("Could not find folder #{metadata_path}") unless File.directory?(metadata_path)
 
           track, release = fetch_track_and_release!(track_name, version_code)
-          UI.user_error!("Unable to find the requested track - '#{Supply.config[:track]}'") unless track
-          UI.user_error!("Could not find release for version code '#{version_code}' to update changelog") unless release
+          UI.user_error!("Unable to find the requested track '#{track_name}' for version code '#{version_code}'") unless track
```

Comment:
Notice I replaced `Supply.config[:track]` with the method argument `track_name` used in the `fetch_track_and_release!` call. From my understanding, the values should be the same at runtime but it nevertheless seems more appropriate to use the one that was used in the call, just in case.